components/dashboard/src/data/organizations/org-members-query.ts

```diff
@@ -26,6 +26,8 @@ export const useOrgMembers = () => {
 
             return publicApiTeamMembersToProtocol(resp.team?.members || []);
         },
+        // If no current org is set, disable query
+        enabled: !!organization,
```

Comment:
**Suggestion (non-blocking):** We can add more context to get more info about this check.

```suggestion
        /**
         * If no current org is set, disable query
         * This is to prevent making a request to the API when there is no organization selected.
         * This is the case when the user is first logging in, or when the user is on the settings page, and no organization is selected.
         */
        enabled: !!organization,
```